modules/op-improving-performance.adoc

```diff
@@ -0,0 +1,19 @@
+// This module is included in the following assemblies:
+// * resource/managing-pipelines-performance.adoc
+
+:_content-type: PROCEDURE
+[id="improving-performance_{context}"]
+= Improving {pipelines-shortname} performance
+
+If you experience slowness or recurrent failures of pipeline runs, you can take any of the following steps to improve the performance of {pipelines-shortname}.
+
+* Increase the number of nodes in the {OCP} cluster on which {pipelines-shortname} runs.
```

Comment:
we need to word this differently so as to NOT draw a direct relation between increasing the nodes and performance improvement - performance will only improve when node resources (CPU, mem) are exhausted and only then the users should consider adding more nodes - this will be done as part of node resource monitoring separately and not much that tekton can help with

publish recommendation for number of nodes
- when running a large number of parallel workloads, resource consumption of nodes can become a bottleneck, hence it should be taken into consideration while observing openshift pipelines’ performance
- if nodes are running out of resources, increasing the number of nodes has shown to have significant impact in the failure rate of pipeline runs.